### ISSUE TITLE
Error messages?

### ISSUE THREAD

**Issue by vineetjc** [vineetjc/pygame-Coin-Fall-#35]
Not sure what this output in the terminal is, hence labelling this as a question:

```
{'reload': <src.ui.input_button.Button instance at 0x7f4a85b420e0>, 'horizontal': <src.ui.input_axis.Axis instance at 0x7f4a85b42320>, 'map': <src.ui.input_button.Button instance at 0x7f4a85b42a28>}
```

I'm assuming it's a debug print statement. @amrit-choudhary most likely from #32

**Comment by amrit-choudhary** [comment]
fixed #37

**Comment by amrit-choudhary** [comment]
issue can be closed
